<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;DejaVu Sans&quot;;color:rgb(174,0,0)</string>
   </property>
   <property name="text">
    <string>Sing in to your existing account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;DejaVu Sans&quot;; color:rgb(174,0,0)</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ph_login">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
   </property>
   <property name="text">
    <string>Login In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_email">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236);
font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_passwd">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236);
font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="mesaj">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="createAccount">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>580</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>550</y>
     <width>221</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't you have an account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ph_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 12pt &quot;DejaVu Sans&quot;;color:rgb(255, 255, 255);
background-color:rgb(174,0,0);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
